riscv/src/runtime.rs

```diff
@@ -148,6 +148,29 @@ impl Runtime {
         self.syscalls.contains_key(&s)
     }
 
+    pub fn with_keccak(mut self) -> Self {
+        self.add_submachine(
+            "std::machines::hash::keccakf::KeccakF",
+            None,
+            "keccakf",
+            [format!(
+                "instr keccakf ~ keccakf.keccakf {};", // operation
+                instr_register_params(0, 2, 2)         // 2 pointers as both input and output
```

Comment:
I think the `keccakf` has 2 inputs and 0 outputs (it won't write to any register, it only modifies memory)

Reply:
Done. Was a bit worried about syntax like `x, y ->` with no output, but guess that's fine.